src/main/java/com/amazonaws/kinesisvideo/producer/KvsRetryStrategy.java

```diff
@@ -0,0 +1,27 @@
+package com.amazonaws.kinesisvideo.producer;
+
+public class KvsRetryStrategy {
+
+    private final long mRetryStrategy;
+    private final long mRetryStrategyConfig;
+    private final KvsRetryStrategyType mKvsRetryStrategyType;
+
+    public KvsRetryStrategy() {
```

Comment:
There are no setters and no constructors to create this object with user-specified values. Is that intended?